### PR TITLE
new image added

### PR DESCRIPTION
**Pull request by barrynormal**
I would like to add this image of crows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## Wise words ##
 
+
+![WHATEVS](/assets/crows.jpeg "crows")
+
```
Comment (mpenf) [inline README.md:5]: does it need cropping more?

Comment (barrynormal) [reply README.md:5]: Um, no

Comment (mpenf) [reply README.md:5]: Let me rephrase that. Please crop the image more tightly on the crows.

Comment (barrynormal) [reply README.md:5]: how is it now?

Comment (mpenf) [reply README.md:5]: better, thanks

**Review by mpenf — APPROVED** [review]
looks good
